src/listener/can_open_listener.py

```diff
@@ -162,6 +179,10 @@ def _sdo_value_changed(self, sdo_index, sdo_value):
             changed = True
         return changed
 
-    def _log_raw_data(self):
+    def _memorize_raw_data(self):
+        # Uses circular buffer implementation.
```

Comment:
misschien iets meer uitleg. Zoals `# Uses circular buffer implementation which overwrites oldest index with new data once the buffer is full`

Reply:
ok dat is goed